lib/ui/collections/album/horizontal_list.dart

```diff
@@ -7,23 +7,22 @@ import "package:photos/events/collection_updated_event.dart";
 import "package:photos/generated/l10n.dart";
 import "package:photos/models/collection_items.dart";
 import "package:photos/theme/ente_theme.dart";
-import "package:photos/ui/collections/collection_item_widget.dart";
+import "package:photos/ui/collections/album/row_item.dart";
 import "package:photos/ui/common/loading_widget.dart";
 
-class AlbumHorizontalListWidget extends StatefulWidget {
+class AlbumHorizontalList extends StatefulWidget {
```

Comment:
Why remove 'Widget' from here? I was following what @vishnukvmd suggested which is to add a 'Widget' postfix if it's a widget till now.

Reply:
To reduce verbosity. System/package widgets also don't always have this word as suffix.